transforms/helpers/EOProp.js

```diff
@@ -220,13 +216,6 @@ class EOProp {
       this.decoratorNames.push("off");
       this.decoratorArgs["off"] = offProperties[name];
     }
-    if (
-      computedProperties.includes(name) &&
```

Comment:
This logic shouldn't be completely removed. We should still be converting the properties to decorators, because the macros still exist, they just don't need the `wrapComputed` anymore.